radixdlt/babylon-wallet-android
# [ABW-3268] Rename link connector

## Description
This PR adds functionality to rename link connectors.
Also minor update in Gateways screen based on zeplin screens.


## How to test

1. Navigate to Linked Connectors screen
2. Edit a link connector by clicking on the pen icon

➡️  Ensure that link connection has not been affected (e.g. terminated or restarted)


## Video

https://github.com/user-attachments/assets/47f61a86-a700-41c6-84b3-94b775c0d197




## PR submission checklist
- [X] I have tested renaming linked connectors
